packages/cli/lib/nango.yaml.schema.v2.json

```diff
@@ -50,6 +50,12 @@
                                             "type": "auto_start must be a boolean."
                                         }
                                     },
+                                    "version": {
+                                        "type": "string",
+                                        "errorMessage": {
+                                            "type": "version must be a string."
+                                        }
```

Comment:
can you add pattern regex so it's consistently enforced?

Reply:
Added

Reply:
there is a small typo interger

Reply:
Thanks, will fix.